pilot/lib/pilot/tester_manager.rb

```diff
@@ -43,6 +40,27 @@ def add_tester(options)
       end
     end
 
+    def add_tester_to_groups(tester: nil, app: nil)
+      default_external_group = app.default_external_group
+      if default_external_group.nil? && config[:groups].nil?
+        UI.error("The app #{app.name} does not have a default external group.")
+        UI.error("Please make sure to pass group names to the `:groups` option.")
+        return
```

Comment:
Should this be `UI.user_error!`? As otherwise we'd show a success message later on I think?